### PR TITLE
Migration to QGIS3.

### PR DESCRIPTION
**Pull request by varmar05**

### CODE REVIEW

File: ConstraintChecker/constraintchecker.py [hunk 1]
```diff
@@ -58,19 +58,18 @@ def __init__(self, iface):
...
         self.freeHandTool = FreehandPolygonMaptool(self.iface.mapCanvas())
-        
-    
+
     def initGui(self):
         # Create action that will start plugin configuration
         self.existingCheckAction = QAction(
-            QIcon(":/plugins/constraintchecker/checker_select_32.png"),
-            u"Check Constraints for Existing Polygon", self.iface.mainWindow())
+            QIcon("/plugins/ConstraintChecker/images/checker_select_32.png"),
```
Comment (DamnBack) [inline ConstraintChecker/constraintchecker.py]: Icon is not showing up on Windows machine.

File: ConstraintChecker/constraintchecker.py [hunk 2]
```diff
@@ -58,19 +58,18 @@ def __init__(self, iface):
...
+            "Check Constraints for Existing Polygon", self.iface.mainWindow())
         self.freehandCheckAction = QAction(
-            QIcon(":/plugins/constraintchecker/checker_freehand_32.png"),
-            u"Check Constraints for Free-hand Polygon", self.iface.mainWindow())
+            QIcon("/plugins/ConstraintChecker/images/checker_freehand_32.png"),
```
Comment (DamnBack) [inline ConstraintChecker/constraintchecker.py]: Icon is not showing up on Windows machine.

File: ConstraintChecker/constraintchecker.py [hunk 3]
```diff
@@ -58,19 +58,18 @@ def __init__(self, iface):
...
+            "Check Constraints for Free-hand Polygon", self.iface.mainWindow())
         self.openConfigurationAction = QAction(
-            QIcon(":/plugins/constraintchecker/checker_config_32.png"),
-            u"Edit Configuration", self.iface.mainWindow())
+            QIcon(":/plugins/ConstraintChecker/images/checker_config_32.png"),
```
Comment (DamnBack) [inline ConstraintChecker/constraintchecker.py]: Icon is not showing up on Windows machine.

**Comment by DamnBack** [comment]
It seems ok now. Ready to be merged.

**Comment by saberraz** [comment]
I don't have the environment set up, so, I am happy if you have tested it. Lets release it and we can fix things in case something is broken!

**Pull request merged**